src/v1/sources/shopify/pixelTransform.js

```diff
@@ -20,6 +29,43 @@ const NO_OPERATION_SUCCESS = {
   },
   statusCode: 200,
 };
+const extractCartToken = (cartToken) => {
```

Comment:
write unit tests for the new functions

Reply:
added test for the single function in place of two earlier